WebRoot/templates/prefs/Pages.template

```diff
@@ -2128,11 +2128,8 @@ NOTE: The tables inside of the filter rule sections MUST contain a tbody element
 		<tr>
 			<td><div><$=ZmMsg.mobileConfigurationDownloadList$></div></td>
 			<td><div id='${id}_configType' tabindex=0></div></td>
-<<<<<<< HEAD
 			<td><div id='${id}_downloadButton' tabindex=0></div></td>
```

Comment:
There shouldn't be two download buttons

Reply:
I only see one button in the UI, what line are we talking about here?

Reply:
Wait see them, hold on.

Reply:
Fixed in https://github.com/Zimbra/zm-web-client/pull/479